src/components/pages/race/reports/content-blocks/report-viz.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react'
+import TableauChart from '~components/charts/tableau'
+import ImageContentBlock from '~components/pages/blog/content-blocks/image-content-block'
+import vizStyles from './report-viz.module.scss'
+
+const Visualization = ({ contentfulId, url, height, vizImage }) => {
+  const tableau = (
+    <TableauChart
+      id={contentfulId}
+      viewUrl={url}
+      className={vizStyles.tableau}
+      height={height + 27} // Account for bottom controls
+    />
+  )
+
+  const image = (
+    <ImageContentBlock
+      image={vizImage}
+      className={vizStyles.image}
+      imageUrl={vizImage.file.url}
+    />
+  )
+
+  const breakpoint = 830
+  const [vizComponent, setVizComponent] = useState(
+    window.innerWidth < breakpoint ? image : tableau,
```

Comment:
@duynguyen158 this breaks SSR because `window` is not set. Let's change this to:

`typeof window !== 'undefined' && window.innerWidth < breakpoint ? image : tableau`

or we can set this state int he `useEffect` hook below.